recipes/bison/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+
+from conans import ConanFile, AutoToolsBuildEnvironment, tools
+from conans.errors import ConanInvalidConfiguration
+import glob
+import os
+import shutil
+
+
+class ConanFileDefault(ConanFile):
+    name = "bison"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://www.gnu.org/software/bison/"
+    description = "Bison is a general-purpose parser generator"
+    topics = ("conan", "bison", "parser")
+    license = "GPL-3.0-or-later"
+    _source_subfolder = "source_subfolder"
+    requires = ("m4/1.4.18", "flex/2.6.4")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"fPIC": [True, False]}
+    default_options = {"fPIC": True}
+
+    _autotools = None
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "bison-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+        if self.settings.os == "Windows":
+            raise ConanInvalidConfiguration("Bison package is not compatible with Windows. Consider using winflexbison instead.")
```

Comment:
The 3.3.2 bincrafters recipe was working with MSVC.
So (with some changes) this 3.5.3 version should also work.

Reply:
bincrafters version requires flex_installer, which was built using mingw. This is why it cannot happen in CCI. Using winflexbison seems the proper solution on windows anyway, right ?

Reply:
Yes, the bincrafters recipe was built using mingw on CI.
But it is also compile-able using MSVC.
I'm working on it atm and will open a pr if I get it to work.